Modules/Microsoft365DSC/DSCResources/MSFT_AADEntitlementManagementAccessPackageAssignmentPolicy/MSFT_AADEntitlementManagementAccessPackageAssignmentPolicy.psm1

```diff
@@ -0,0 +1,1559 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.String]
+        $Id,
+
+        [Parameter()]
+        [System.String]
+        $AccessPackageId,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $AccessReviewSettings,
+
+        [Parameter()]
+        [System.Boolean]
+        $CanExtend,
+
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.Int32]
+        $DurationInDays,
+
+        [Parameter()]
+        [system.datetimeoffset]
+        $ExpirationDateTime,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $Questions,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $RequestApprovalSettings,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $RequestorSettings,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $CustomExtensionHandlers,
+
+        [Parameter(Mandatory = $true)]
```

Comment:
Like other PR, the Ensure parameter should not be mandatory. But the Display Name instead.